x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -389,6 +390,76 @@ func TestInput(t *testing.T) {
 			handler:  defaultHandler("GET", ""),
 			expected: []string{`{"hello":[{"world":"moon"},{"space":[{"cake":"pumpkin"}]}]}`},
 		},
+		{
+			name: "Test simple Chain GET request",
```

Comment:
would be nice to add non trivial tests too (multiple chains, use of cursor, etc) to ensure the whole new feature set works as expected